hook-common/src/pgqueue.rs

```diff
@@ -511,12 +555,10 @@ impl PgQueue {
         queue_name: &str,
         table_name: &str,
         url: &str,
-        worker_name: &str,
         retry_policy: RetryPolicy,
     ) -> PgQueueResult<Self> {
         let name = queue_name.to_owned();
         let table = table_name.to_owned();
-        let worker = worker_name.to_owned();
         let pool = PgPoolOptions::new()
```

Comment:
Yeah, pretty much. We are currently not configuring the connection pool at all, as this will have to be synced with our Postgres instance. I can add the configuration keys, and we can turn those knobs over time.